subprojects/android-test/test-report/src/main/kotlin/com/avito/android/test/report/transport/OutputFileProvider.kt

```diff
@@ -0,0 +1,36 @@
+package com.avito.android.test.report.transport
+
+import androidx.test.platform.app.InstrumentationRegistry
+import java.io.File
+
+internal class OutputFileProvider {
+
+    sealed class Result {
```

Comment:
`kotlin.Result` ?

Reply:
It's usage disabled bu default, i think we should decide it's usage outside of this PR context